db/migrate/20210420135208_add_name_tsvector_index_to_users.rb

```diff
@@ -0,0 +1,17 @@
+class AddNameTsvectorIndexToUsers < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    add_index(
+      :users,
+      "to_tsvector('simple'::regconfig, COALESCE((name)::text, ''::text))",
```

Comment:
this is needed so that search on `users.name` is as fast as `users.username`. This is what we'll need to make https://github.com/forem/rfcs/issues/194 work as well